README.md

```diff
@@ -1,47 +1,93 @@
 # PocketSVG
 
-A simple toolkit for using displaying and manipulating SVGs on iOS and OS X.
+[![Build Status](https://travis-ci.org/arielelkin/PocketSVG.svg?branch=master)](https://travis-ci.org/arielelkin/PocketSVG)
+
+A simple toolkit for displaying and manipulating SVGs on iOS and macOS.
 
 Convert your SVGs into 
```

Comment:
Nope, it's crucial for a readme file to include a concise project description at the top. See for examples: https://github.com/AFNetworking/AFNetworking and https://github.com/matiassingers/awesome-readme

You're right to point out that our SVG processing involves a conversion into UI/NSBezierPath, and the reason I didn't mention that that is because I think that the majority of our users will want a simple SVG --> UI/NSView utility. But the bullet points that follow should make it clear that we also support SVG-->BezierPath.

and yep, no such thing as NSBezierCurve: https://github.com/arielelkin/PocketSVG/pull/54/commits/7d80b8ebe19162d769dde93d1584ac6aed218373